pom.xml

```diff
@@ -11,7 +11,8 @@
   <version>1.11.2-1-SNAPSHOT</version>
   <packaging>hpi</packaging>
   <name>Jenkins jQuery plugin</name>
-  <url>http://wiki.jenkins-ci.org/display/JENKINS/jQuery+Plugin</url>
+  <description>This plugin is a library plugin for other plugins to share common jQuery. It also allows users to use jQuery on each view descriptions.</description>
```

Comment:
@recena The description has been taken from the wiki, where it was originally written by the creator and maintainers of the plugin, I believe it makes sense to be coherent with the wiki.

Regarding the name, if we change the value in pom is reflected in jenkins UI, potentially making confusion on users who expect a `Jenkins jQuery Plugin`. I have search the jenkinsci repo and I have found two different patterns for name `Jenkins X plugin`  and `X plugin`given that the current name pattern is shared by several plugins, for example `ssh-slaves` or `subversion`, that is the existing name pattern is not an exception. Do you think change the name is worth the possible confusion?